sensu/config.go

```diff
@@ -70,6 +70,10 @@ func NewConfigFromFile(flagset *configFlagSet, configFile string) (*Config, erro
 
 	if cfg.Client().Name == "" {
 		return nil, errNoClientName
+	} else {
+		// if we get a Name, let's also add the default subscription - client:name
+		// without at least one subscription sensu server will crash
+		cfg.Client().Subscriptions = append(cfg.Client().Subscriptions, "client:"+cfg.Client().Name)
```

Comment:
PS: because you already added this subscription in the `Client()` method, you don't need to add it again here